docs/tutorials/best-practices/error-safety.md

```diff
@@ -91,7 +91,7 @@ cause a fatal error that stops your program from running.
 	print("before")
 	print("before")
 
-	scope:Computed(function()
+	scope:Computed(function(use, scope)
```

Comment:
Use and Scope shouldn't be provided as it is never used
```suggestion
	scope:Computed(function()
```

Reply:
reverted